2023-10-12/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/vmorsell/joel/2023-10-12/names"
+)
+
+func main() {
+	fmt.Println(names.GitKenobi)
```

Comment:
function from package fmt, Println, referring to file names, variable gitkenobi

Reply:
Calling the exported function Println from the package _fmt_. First and only argument is the exported variable GitKenobi from the package _names_